lib/service/logger.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2016 Kinvey Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+const isNil = require('lodash.isnil');
+
+function _log(level, message) {
+  if (isNil(message)) return;
+  if (message instanceof Error) message = message.toString();
+
+  const logMessage = JSON.stringify({
+    message,
+    level,
+    backingServerId: process.env.BACKING_SERVER_ID
```

Comment:
@galniv noted. I'll revise the kubernetes pod structure to also include the relevant `datalink._id` on the PATH.